store/etcd/v3/etcd.go

```diff
@@ -171,6 +171,9 @@ func (s *EtcdV3) Put(key string, value []byte, opts *store.WriteOptions) (err er
 // Delete a value at "key"
 func (s *EtcdV3) Delete(key string) error {
 	resp, err := s.client.KV.Delete(context.Background(), s.normalize(key))
+	if resp == nil {
```

Comment:
Without checking the client code, does `err` have a value allways when the `resp` is `nil`?

Reply:
It difficult to say, I changed the algorithm.